HW3/forms.py

```diff
@@ -0,0 +1,22 @@
+# WTForms
+
+from flask_wtf import FlaskForm
+from wtforms import StringField, IntegerField, SelectField, PasswordField, EmailField
+from wtforms.validators import DataRequired, Length, EqualTo
+
+
+class LoginForm(FlaskForm):
+    # validators может принимать несколько значений-проверок. это список.
+    name = StringField('Username', validators=[DataRequired()])
+
+    password = PasswordField('Password', validators=[DataRequired()])
+
+
+
+
+
+class RegisterForm(FlaskForm):
+    name = StringField('Name', validators=[DataRequired()])
+    email = EmailField('Email', validators=[DataRequired()])
+    password = PasswordField('Password', validators=[DataRequired(), Length(min=6)])
+    confirm_pas = PasswordField('Password', validators=[DataRequired(), EqualTo('password')])
```

Comment:
Не помещает добавить поле `SubmitField`
```suggestion
    confirm_pas = PasswordField('Password', validators=[DataRequired(), EqualTo('password')])
    submit = SubmitField('Зарегистрироваться')
```